lms/envs/common.py

```diff
@@ -1860,6 +1860,9 @@
     # Microsite configuration
     'microsite_configuration',
 
+    # RSS Proxy
+    'rss_proxy',
```

Comment:
I think the jury is still out on that approach. Let's make it a priority to figure out a good rule of thumb on our code organization philosophy. I'd rather not hold up this work on that decision. We can always reorganize later.